### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Filter posts by all selected tags

### DISCUSSION THREAD

**Discussion by Rudomitori** [dailydotdev/daily#1565] [Feature Requests]
### Concise Description

Filtered posts must includes all selected tags - not any ones

### Description

I've tried to create a custom feed about backend on rust by adding tags `rust` and `backend`
and in the result I received:
* How to make your Express app 9x faster (and type-safe)
  `performance` `nodejs` **`backend`** `typescript`
* Tinybird vs. ClickHouse
  `cloud` **`backend`** `real-time-analytics` `clickhouse` `tinybird`
* Ultimate VS Code setup for Rust development (2025)
  `devtools` **`rust`** `vscode`
* And many other not relevant posts

As I've understand, custom feeds contain all posts that have at least one of the selected tags.
This filtering logic doesn't enable me to create a custom feed, that will be useful for me

### Solution Proposed

I propose to change the filtering logic so that custom feeds contain only posts that have **all** selected tags

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @Rudomitori,

That's a very interesting use case. We will consider how to approach it
